CMakeLists.txt

```diff
@@ -1442,10 +1442,20 @@ if(BUILD_TESTS)
       NAME pi_basic
       PYTHON_SCRIPT ${CMAKE_CURRENT_LIST_DIR}/tests/infra/basicperf.py
       CLIENT_BIN ./submit
-      ADDITIONAL_ARGS --package "samples/apps/basic/libbasic"
-                      --max-writes-ahead 1000 --repetitions 10000
+      ADDITIONAL_ARGS --package "samples/apps/basic/libbasic" --repetitions
```

Comment:
Why no write-ahead cap?

Reply:
Moved to the script as I removed one layer of command overrides. We allowed a mix of client and nodes overrides, which I think is very error prone.